tor/core/inbox.py

```diff
@@ -72,11 +72,25 @@ def process_reply(reply: Comment, cfg: Config) -> None:
 
         if "image transcription" in r_body or is_comment_transcription(reply, cfg):
             post_link = reply.submission.url
+            submission = reply.submission
+            username = reply.author.name
             sub_name = extract_sub_from_url(post_link)
+            user_url = i18n["urls"]["reddit_url"].format(f"/u/{username}")
+            post_url = submission.url
+            tor_post = submission.tor_url
```

Comment:
we're working with a PRAW object, so there isn't going to be a `tor_url` attribute here.